app/src/components/v-form/v-form.vue

```diff
@@ -339,6 +339,11 @@ export default defineComponent({
 			}
 
 			function setPrimaryKeyReadonly(field: Field) {
+				// Allow to manually set auto incremented field if it is required, might still be disabled
+				if (props.primaryKey === '+' && field.schema?.has_auto_increment === true && field.meta?.required) {
+					return field;
+				}
```

Comment:
Instead of returning early here, we should include this check in the existing if statement below

Reply:
I tried that initially, but it becomes quite a big if statement. Will try again though, since we now can just look at the readonly property and not any more at the `has_auto_increment`